src/input.rs

```diff
@@ -335,7 +384,7 @@ impl<'a> Input<'a> {
         out.set_color(&self.theme.title)?;
         write!(out, " {}", self.title)?;
         out.set_color(&self.theme.selected_option)?;
-        writeln!(out, " {}", &self.render_input()?.to_string())?;
+        writeln!(out, "{}", self.input)?;
```

Comment:
We should keep that space otherwise result is printed out to close.
<img width="525" alt="Screenshot 2024-04-27 at 09 13 14" src="https://github.com/jdx/demand/assets/181773/422feff2-3768-406c-a3d5-68d6b50d289a">

Reply:
oops